lightning/src/ln/channel.rs

```diff
@@ -1986,14 +1986,21 @@ impl<Signer: Sign> Channel<Signer> {
 		self.counterparty_selected_channel_reserve_satoshis)
 	}
 
-	// Get the fee cost of a commitment tx with a given number of HTLC outputs.
+	// Get the fee cost in MSATS of a commitment tx with a given number of HTLC outputs.
 	// Note that num_htlcs should not include dust HTLCs.
 	fn commit_tx_fee_msat(&self, num_htlcs: usize) -> u64 {
 		// Note that we need to divide before multiplying to round properly,
 		// since the lowest denomination of bitcoin on-chain is the satoshi.
 		(COMMITMENT_TX_BASE_WEIGHT + num_htlcs as u64 * COMMITMENT_TX_WEIGHT_PER_HTLC) * self.feerate_per_kw as u64 / 1000 * 1000
 	}
 
+	// Get the fee cost in SATS of a commitment tx with a given number of HTLC outputs.
+	// Note that num_htlcs should not include dust HTLCs.
+	#[inline]
+	fn commit_tx_fee_sat(feerate_per_kw: u32, num_htlcs: usize) -> u64 {
+		feerate_per_kw as u32 * (COMMITMENT_TX_BASE_WEIGHT + num_htlcs as u64 * COMMITMENT_TX_WEIGHT_PER_HTLC) / 1000
```

Comment:
nit: would prefer if this function called `commit_tx_fee_msat` and divided by 1000

Reply:
Though I would need to make it as a method and as such reduce the convenience for future usages.